### ISSUE TITLE
Quartz fiber recipe

### ISSUE THREAD

**Issue by Sharkum** [FTBTeam/FTB-Interactions#1212]
## Modpack Version: 
2.0.9

## Suggestions:
Why does the quartz fibre craft need 144mb of glass. Glass only has only 375 and 1000 mb recipes in the fluid extractor. I know this might seem nitpicky but could you increase it to 1000mb or something more manageable. There are only 2 recipes using glass in the assembler so it doesnt make sense to make an assembler which is kept in stock with glass. Same goes for the quartz glass.🙌😢

**Comment by Sharkum** [comment]
If i make a recipe for the fibers, it gives me 1000 fibers for 18000 mb of glass which is 125 sets. Any less and there is some glass left over in the assembler. I use a melter assembler setup to craft theese btw.

**Comment by Sharkum** [comment]
I closed it originally because I discovered volumetric flasks were a thing. But once I used them I realised that the problem was with the ratio of sand to glass. Just increase the glass requirement to 200mb maybe for the recipes. Otherwise you just have to keep glass in stock for the crafts. If there were more than two recipes then it would have been fine but it does feel too much work to setup a melter separately for glass.
